test/jdk/java/net/ProxySelector/LoopbackAddresses.java

```diff
@@ -151,3 +153,18 @@ public int getPort() {
         }
     }
 }
+
+class LoopbackAddressesHandler implements HttpHandler {
+
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        try {
+            exchange.sendResponseHeaders(200, 0);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        try(PrintWriter pw = new PrintWriter(exchange.getResponseBody())) {
+            pw.print("Hello .");
+        }
```

Comment:
I know that now UTF-8 is supposed to be the default - but I'd prefer to either make it explicit, or add a comment stating that since Java 18 PrintWriter will use UTF-8 encoding by default.